pkg/kn/commands/plugin/plugin_verifier_windows.go

```diff
@@ -0,0 +1,190 @@
+// Copyright © 2019 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build windows
```

Comment:
builds only for Windows platform

Reply:
This was redundant, we should either use file suffix or a build tag denoting the choice of platform this file should compile on. We're using file suffix.